src/main/java/de/hpi/bpt/argos/api/entity/EntityEndpoint.java

```diff
@@ -127,24 +132,29 @@ static String getEntityUri(long entityId) {
     /**
      * This method returns the basic URI to retrieve the event types of an entity.
      * @param entityId - the id of the entity to be searched for
+	 * @param includeChildEvents - indicates, whether child events should be considered
      * @return - the URI to retrieve event types from
      */
-    static String getEventTypesOfEntityUri(long entityId) {
-        return  getEventTypesOfEntityBaseUri().replaceAll(getEntityIdParameter(true), Objects.toString(entityId, "0"));
+    static String getEventTypesOfEntityUri(long entityId, boolean includeChildEvents) {
+        return  getEventTypesOfEntityBaseUri()
+				.replaceAll(getEntityIdParameter(true), Objects.toString(entityId, "0"))
+				.replaceAll(getIndexToParameter(true), Objects.toString(includeChildEvents, "false"));

```

Comment:
You're replacing the index parameter instead of the includeChildEvents parameter.